rotkehlchen/chain/evm/decoding/hop/balances.py

```diff
@@ -0,0 +1,165 @@
+                continue
+
+            try:
+                staking_token, rewards_token = EvmToken(evm_address_to_identifier(
+                    address=checksummed_staking_token,
+                    chain_id=self.evm_inquirer.chain_id,
+                    token_type=EvmTokenKind.ERC20,
+                )), EvmToken(evm_address_to_identifier(
+                    address=checksummed_rewards_token,
+                    chain_id=self.evm_inquirer.chain_id,
+                    token_type=EvmTokenKind.ERC20,
+                ))
+            except (UnknownAsset, WrongAssetType):
+                log.warning(f'Found hop balance for unknown token {checksummed_staking_token} or {checksummed_rewards_token}. Skipping')  # noqa: E501
+                continue
+
+            staked_rewards = self.evm_inquirer.multicall(
+                calls=[(
+                    staking_contract.address,
+                    staking_contract.encode(method_name='earned', arguments=[user]),
+                ) for user in addresses],
+                call_order=call_order,
+                calls_chunk_size=chunk_size,
+            )
+            token_price = Inquirer.find_usd_price(staking_token)
+            rewards_price = Inquirer.find_usd_price(rewards_token)
+            for user, lp, reward in zip(addresses, staked_lps, staked_rewards, strict=True):
+                try:
+                    if (balance := staking_contract.decode(
+                        result=lp,
+                        method_name='balanceOf',
+                        arguments=[user],
+                    )[0]) != ZERO and (balance_norm := token_normalized_value(
+                        token_amount=balance,
+                        token=staking_token,
+                    )) > ZERO:
+                        balances[user].assets[staking_token] += Balance(
+                            amount=balance_norm,
+                            usd_value=token_price * balance_norm,
+                        )
+                except DeserializationError:
+                    log.warning(f'Failed to decode staked balance for {user}. Skipping')
+                    continue
+
+                try:
+                    if (rewards := staking_contract.decode(
+                        result=reward,
+                        method_name='earned',
+                        arguments=[user],
+                    )[0]) != ZERO and (reward_norm := token_normalized_value(
+                        token_amount=rewards,
+                        token=rewards_token,
+                    )) > ZERO:
+                        balances[user].assets[rewards_token] += Balance(
+                            amount=reward_norm,
+                            usd_value=rewards_price * reward_norm,
+                        )
+                except DeserializationError:
+                    log.warning(f'Failed to decode earned rewards for {user}. Skipping')
+                    continue
```

Comment:
Also a question for all those logs. Should they be warnings or errors. Are they ever supposed to happen? If not then this should be an error.

Reply:
Only the `No staked balances` scenario is expected, changed the rest to `log.error`.